pulseaudio-control.bash

```diff
@@ -337,6 +339,8 @@ Options: [defaults]
                                         [$OSD]
   --icon-muted <icon>                   icon to use when muted [none]
   --icon-sink <icon>                    icon to use for sink [none]
+  --format <string>                     use a format string to control the output
```

Comment:
I think we should list the available variables that can be used in this format. Something along the lines of "Available variables: $VOL_ICON, ..."